packages/perseus/src/components/i18n-context.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * PerseusI18nContext provides a way to set the strings and locale that are
+ * used inside the Math Input package.
```

Comment:
```suggestion
 * used inside this package.
```

(or `inside the Perseus package`)